test/tsconfig.json

```diff
@@ -6,6 +6,17 @@
     "files": [
         "utils/mocks.ts",
         "utils/MochaLoader.ts",
-        "AudioPlayr/_.ts"
+        "AudioPlayr/addEventImmediate.ts",
+        "AudioPlayr/addEventListener.ts",
+        "AudioPlayr/clearTheme.ts",
+        "AudioPlayr/pauseSound.ts",
+        "AudioPlayr/play.ts",
+        "AudioPlayr/playLocal.ts",
+        "AudioPlayr/playSound.ts",
+        "AudioPlayr/playTheme.ts",
+        "AudioPlayr/removeEventListeners.ts",
+        "AudioPlayr/setMutedOff.ts",
+        "AudioPlayr/setMutedOn.ts",
+        "AudioPlayr/soundStop.ts"
```

Comment:
Missing tests for `pause` and `pauseAll`. `soundStop` is private and so shouldn't be tested.